src/lib.rs

```diff
@@ -330,9 +330,9 @@ impl EmojiLabel {
             ui.spacing_mut().item_spacing.x = 0.0;
             match segment {
                 TextSegment::Text(text) => {
-                    let mut label = egui::Label::new(text.clone()).truncate(self.truncate);
+                    let mut label = egui::Label::new(text.clone()).truncate();
                     if let Some(wrap) = self.wrap {
-                        label = label.wrap(wrap);
+                        label = label.wrap();
```

Comment:
Same thing here. The `wrap()` call does not respect the value of `self.wrap`. `EmojiLabel`'s API needs to be changed to match this.

Reply:
```
    fn show_segments(&self, ui: &mut egui::Ui, state: &mut LabelState) -> egui::Response {
        let mut resp = empty_response(ui.ctx().clone());
        let font_height = ui.text_style_height(&egui::TextStyle::Body);

        for segment in &state.segments {
            ui.spacing_mut().item_spacing.x = 0.0;
            match segment {
                TextSegment::Text(text) => {
                    let mut label = egui::Label::new(text.clone());
                    if let Some(truncate) = self.truncate {
                        if truncate
                        {
                            label = label.truncate();
                        }
                    }
                    if let Some(wrap) = self.wrap {
                        if wrap
                        {
                            label = label.wrap();
                        }
                    }
                    if let Some(selectable) = self.selectable {
                        label = label.selectable(selectable);
                    }
                    if let Some(sense) = self.sense {
                        label = label.sense(sense);
                    }
                    resp |= ui.add(label);
                }
                TextSegment::Emoji(emoji) => {
                    let Some(source) = get_source_for_emoji(emoji) else {
                        continue;
                    };

                    let image_rect = ui
                        .add(
                            egui::Image::new(source)
                                .fit_to_exact_size(egui::vec2(font_height, font_height)),
                        )
                        .rect;

                    // for emoji selection and copying:
                    resp |= ui.put(
                        image_rect,
                        egui::Label::new(RichText::new(emoji).color(egui::Color32::TRANSPARENT)),
                    );
                }
            }
        }
        resp
    }
```
Could it be a way to do this, I am really unsure.